src/data-fetcher/bigwig/higlass-bigwig-datafetcher.ts

```diff
@@ -0,0 +1,323 @@
+/* TODO: Support esm
```

Comment:
This was an error. Let's remove the line.

```suggestion
/*
```